test/utils/probability/test_lin_ess.py

```diff
@@ -373,15 +373,12 @@ def test_multivariate(self):
                 check_feasibility=True,
             )
             nu = torch.full((d, 1), lower_bound + 2, **tkwargs)
-            theta_active = sampler._find_active_intersections(nu)
-            self.assertTrue(
-                torch.equal(theta_active, sampler._full_angular_range.view(-1))
-            )
-            rot_angle, slices = sampler._find_rotated_intersections(nu)
-            self.assertEqual(rot_angle, 0.0)
-            self.assertAllClose(
-                slices, torch.tensor([[0.0, 2 * torch.pi]], **tkwargs), atol=atol
-            )
+
+            # Get the left and right endpoints of the active intervals.
+            # As the entire ellipse is feasible, the cumulative length should be 2 * pi.
+            left, right = sampler._find_active_intersection_angles(nu)
+            csum = right.sub(left).clamp(min=0.0).cumsum(dim=-1)
```

Comment:
I'm not sure if this works, but I'd find it clearer because I had to look up `torch.sub` . Also, why clamp?

```suggestion
            csum = (right - left).clamp(min=0.0).cumsum(dim=-1)
```

Reply:
Yes, this also works.

The clamping is needed because not all entries in `left` and `right` are active intersection angles. A pair of angles `left[i]` and `right[j]` are active if `left[i] <= right[i]`. In this case, the interval `[left[i], right[i]]` is an active interval. Otherwise, `left[i]` and `right[j]` are dummy angles that do not contribute to the cumulative length. Therefore, we have to clamp them.